src/main/java/seedu/storemando/logic/parser/ReminderCommandParser.java

```diff
@@ -0,0 +1,61 @@
+package seedu.storemando.logic.parser;
+
+import static seedu.storemando.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.storemando.logic.commands.ReminderCommand;
+import seedu.storemando.logic.parser.exceptions.ParseException;
+import seedu.storemando.model.item.ItemExpiringPredicate;
+
+/**
+ * Parses input arguments and creates a new ReminderCommand object
+ */
+public class ReminderCommandParser implements Parser<ReminderCommand> {
+    /**
+     * Convert the given number and the time unit to the number of days.
+     * @param num The number use to covert to days
+     * @param timeUnit The time unit in terms of days and weeks
+     * @return The number of days
+     * @throws ParseException if the user input does not conform the expected keyword
+     */
+    private long timeConversation(long num, String timeUnit) throws ParseException {
+        switch (timeUnit) {
```

Comment:
I'm assuming this is not case-sensitive. Does it mean that "Weeks" would not match "weeks"? If that is the case, we have to clearly specify this in the user guide.

Reply:
u could try using equalsIgnoreCase() like I did for sortCommand

Reply:
All right! I will make it case-insensitive.